src/scripts/UI/components/currentlySelectedSectionComponent.tsx

```diff
@@ -6,18 +6,19 @@ interface CurrentlySelectedSectionProps {
 	onSectionLocationContainerClicked: any;
 	textToDisplay: string;
 	tabIndex: number;
+	expanded: boolean;
 }
 
 class CurrentlySelectedSectionClass extends ComponentBase<{}, CurrentlySelectedSectionProps> {
 	render() {
 		return (
 			<div id={Constants.Ids.saveToLocationContainer} className="SaveToLocationContainer" >
 				<a id={Constants.Ids.sectionLocationContainer} role="button" style="outline-style: none;"
-					{...this.enableInvoke(this.props.onSectionLocationContainerClicked, this.props.tabIndex)}>
+					{...this.enableInvoke(this.props.onSectionLocationContainerClicked, this.props.tabIndex)} aria-expanded={this.props.expanded}>
 					<div className="OpenSectionPickerArrow">
 						<img className="arrowDown" src={Utils.getImageResourceUrl("dropdown_arrow.png")} />
 					</div>
-					<div className="SectionLocation" title={this.props.textToDisplay}>{this.props.textToDisplay}</div>
```

Comment:
Getting rid of `title` removes mouseover, which makes it hard to read the entire path